Formula/hercules.rb

```diff
@@ -21,15 +21,21 @@ class Hercules < Formula
 
   head do
     url "https://github.com/hercules-390/hyperion.git"
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
   skip_clean :la
 
   def install
-    system "./autogen.sh" if build.head?
+    if build.head?
+      system "./autogen.sh"
+    else
+      system "autoreconf", "-fvi"
+    end
```

Comment:
I think this should be reported upstream and only used for ARM based builds

Reply:
I'll add this, I have seen other formula that do `autoreconf` always.
I figured this was ok, but adding if arm on.

Reply:
Well, sometimes it's always needed. But here it's only to fix arm builds.